### PR TITLE
Add license headers for GeoTools Shapefile classes

### PR DESCRIPTION
**Pull request by dr-jts**
Signed-off-by: dr-jts <mtnclimb@gmail.com>

### CODE REVIEW

File: modules/app/src/main/java/org/locationtech/jtstest/testbuilder/io/shapefile/Shapefile.java [hunk 1]
```diff
@@ -1,3 +1,23 @@
+/*
+ * Copyright (c) 2003 Open Source Geospatial Foundation, All rights reserved.
+ * 
+ * This program and the accompanying materials are made available under the terms
+ * of the OSGeo BSD License v1.0 available at:
+ *
+ * https://www.osgeo.org/sites/osgeo.org/files/Page/osgeo-bsd-license.txt
+ */
+/*
```
Comment (dr-jts) [inline modules/app/src/main/java/org/locationtech/jtstest/testbuilder/io/shapefile/Shapefile.java]: Dang, got picked the wrong one.  Ok as it stands?

Comment (dr-jts) [inline modules/app/src/main/java/org/locationtech/jtstest/testbuilder/io/shapefile/Shapefile.java]: My reasoning was that this code is primarily GeoTools, with mods by Vivid.  But can see the logic in date order as well.

Comment (jodygarnett) [inline modules/app/src/main/java/org/locationtech/jtstest/testbuilder/io/shapefile/Shapefile.java]: yep, just hard to follow, principle of least surprise.
